File: CursoPOOUber/Csharp/Cash.cs
using PaymentTypes;

namespace PaymentTypes.Cash 
{

    //Definition of the "Cash" class, which inherits from "Cash" class.
    public class Cash : Payment
    {

        public Cash(int id) : base(id)
        {
            this.Id = id;
        }

        public void PrintCashData()
        {
            Console.WriteLine("This user has paid in cash.");
        }
        
    }
}

File: CursoPOOUber/Java/Cash.java
package CursoPOOUber.Java;

//DEfinition of the "Cash" class, which inherits from "Payment" class.
class Cash extends Payment {
    
    public Cash(Integer id){
        super(id);
        this.id = id;
    }

    void printCashData() {
        System.out.println("This user has paid in cash.");
    }
}
